_install-and-configure/plugins.md

```diff
@@ -2,29 +2,34 @@
 layout: default
 title: Installing plugins
 nav_order: 90
+has_children: true
 redirect_from:
    - /opensearch/install/plugins/
    - /install-and-configure/install-opensearch/plugins/
 ---
 
 # Installing plugins
 
-You can install individual plugins for OpenSearch based on your needs. For information about available plugins, see [Available plugins](#available-plugins).
+OpenSearch is comprised of a number of plugins that add features and capabilities to the core platform. The plugins that are available to you are dependent on how OpenSearch was installed and which plugins were subsequently added or removed. For example, the minimal distribution option of OpenSearch enables only core functionality, such as search. Using the minimal distribution of OpenSearch is beneficial when working in a testing environment, you have custom plugins, or are intending to integrate OpenSearch with other services.  
 
-For plugins to work properly with OpenSearch, all plugins must have the ability to access the data in the cluster, including metadata about cluster operations. Therefore, to protect your cluster's data and preserve its integrity, first be sure you understand the function of a plugin before installing it on your OpenSearch cluster. Second, when selecting a custom plugin, make sure the plugin's source is a reliable one.
+The standard distribution of OpenSearch has a lot more functionality bundled in. You can choose to add additional plugins or remove any of the plugins you don't need. 
+
+For a sampling of the available plugins, see [Available plugins](#available-plugins).
+
+For a plugin to work properly with OpenSearch, it must have the ability to access data and metadata in the cluster. Consequently, to safeguard your cluster's data integrity, it is crucial that you understand a plugin's functionality before installation. Additionally, when opting for a community-provided plugin, ensure that the source is trustworthy and reliable.
```

Comment:
wdyt? `For a plugin to work properly with OpenSearch, it may request certain permissions as part of the installation process. Please review the requested permissions and proceed accordingly.`

Reply:
Sounds good!